src/components/ExportGithub.tsx

```diff
@@ -177,11 +192,37 @@ export const ExportGithub = ({
           icon="key"
           intent="primary"
           onClick={async () => {
+            const params = isDev
+              ? `client_id=Iv1.4bf062a6c6636672&state=${state}`
+              : `client_id=Iv1.e7e282a385b7b2da&state=${state}`;
             authWindow.current = window.open(
-              "https://github.com/login/oauth/authorize?client_id=Iv1.e7e282a385b7b2da",
+              `https://github.com/login/oauth/authorize?${params}`,
               "_blank",
               `width=${WINDOW_WIDTH}, height=${WINDOW_HEIGHT}, top=${WINDOW_TOP}, left=${WINDOW_LEFT}`
             );
+
+            let attemptCount = 0;
+            const check = () => {
+              if (attemptCount < 10) {
```

Comment:
let's do two min

Reply:
Two minutes seems excessive.  To be clear, this would mean the user would wait 2 minutes staring at "Please Wait" (if they didn't manually cancel) and we would send 120 `POST` requests.

What scenario would waiting two minutes be useful for?

Reply:
30 seconds then? just trying to mitigate any scenario with spotty internet/slow devices, 10 seconds felt quick

Reply:
Sold!